lib/pkg.ml

```diff
@@ -16,7 +16,7 @@ let uri_domain uri =
 let uri_sld uri =
   match uri_domain uri with _ :: sld :: _ -> Some sld | _ -> None
 
-let uri_append u s =
+let uri_append s u =
```

Comment:
I think this would be the right time to name one argument as soon as the argument order becomes non trivial

Reply:
Done. Thanks!